docs/reference/commandline/volume_create.md

```diff
@@ -10,20 +10,19 @@ parent = "smn_cli"
 
 # volume create
 
-    Usage: docker volume create [OPTIONS]
+    Usage: docker volume create [VOLUME] [OPTIONS]
```

Comment:
this is inconsistent with the declaration in the cobra command.

`create [VOLUME] [OPTIONS]`
&
`create [OPTIONS] [VOLUME]`

I know it doesn't matter, but consistency would be nice.